source/cloud-native-platform/path-to-live/public-dns.html.md.erb

```diff
@@ -42,3 +42,12 @@ Where:
 This should be the `name` attribute from env.tfvars file e.g. [name attribute](https://github.com/hmcts/azure-platform-terraform/blob/fa598ca88221555abcbe54a52f0738ca20d48f59/environments/prod.tfvars#L17)
 
 Send the PR for review to the [Platform Operations](https://hmcts-reform.slack.com/app_redirect?channel=platops-code-review) team. Once the PR is merged terraform will create appropriate records.
+
+
+### Synchronizing Public DNS Zone CNAME Records with Private DNS Zone
+
+We synchronize CNAME records from our public DNS zone to the corresponding private DNS zone, such as `sandbox.platform.hmcts.net`. This ensures that internal services can reach the correct destinations without encountering failures.
+
+If you prefer not to sync specific public CNAME records to the private DNS zone, you can easily achieve this by setting `syncPrivateDNS` to `false`. By default, this setting is `true`, simplifying the synchronization process when needed. Please note that the pipeline responsible for syncing records is scheduled to run overnight.
+
+[Refer to this example](https://github.com/hmcts/azure-public-dns/blob/0295fff5f4e498b9f2acf564e9f656cf4f1dbb34/environments/sandbox.yml#L259) for guidance on excluding records from synchronization.
```

Comment:
I'd maybe make it clear that if you had been syncing and you now want separate records, update the azure-private-dns repo. Also are we still doing CNAMEs only?

Reply:
correct CNAME only - I have updated the docs now :)